docs/bokeh/source/docs/user_guide/output/jupyter.rst

```diff
@@ -346,7 +346,7 @@ The main `Bokeh`_ repository also includes some notebook comms examples:
 * :bokeh-tree:`examples/output/jupyter/push_notebook/Jupyter Interactors.ipynb`
 * :bokeh-tree:`examples/output/jupyter/push_notebook/Numba Image Example.ipynb`
 
-.. _bokeh-notebook: https://github.com/bokeh/bokeh-notebooks
+.. _bokeh-tutorial: https://github.com/bokeh/tutorial/
 .. _Binder: https://mybinder.org/v2/gh/bokeh/bokeh-notebooks/HEAD?labpath=index.iynb
```

Comment:
Here is one link to mybinder where I don't know how to replace it. Should I delete this linke and the corresponding explenation?